packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AgentChatAssistantBubble.tsx

```diff
@@ -56,20 +56,24 @@ const AssistantBubbleContent: FC<{
             : isStubbed // avoid brief flash of `THINKING_STUB`
             ? ''
             : message.message ?? 'No response...';
+
     return (
         <>
-            <AiChartToolCalls
-                toolCalls={streamingState?.toolCalls ?? message.toolCalls}
-                type={
-                    streamingState
-                        ? isStreaming
-                            ? 'streaming'
-                            : 'finished-streaming'
-                        : 'persisted'
-                }
-                metricQuery={metricQuery}
-                projectUuid={projectUuid}
-            />
+            {isStreaming && (
+                <AiChartToolCalls
+                    toolCalls={streamingState?.toolCalls}
+                    type="streaming"
+                />
+            )}
+
+            {!isStreaming && message.toolCalls.length > 0 && (
+                <AiChartToolCalls
+                    toolCalls={message.toolCalls}
+                    type="persisted"
```

Comment:
is there a way we could have the `finished-streaming` state back? this way we can have the dropdown open when the stream finishes.
My idea was to:
- open when streaming
- open when finished response
- closed when revisiting threads